css/style.css

```diff
@@ -0,0 +1,775 @@
+  margin: 20px 30% 20px 30%;
+  padding-left: 20px;
+}
+
+.author > a {
+  font-size: 0.9em;
+}
+
+.social > a {
+  background-color: rgb(245, 239, 239);
+  padding: 6px;
+  border-radius: 50%;
+}
+
+.main-w-sidebar {
+  width: 70%;
+  margin-left: 30%;
+}
+
+.main-w {
+  float: left;
+  width: 55%;
+}
+
+.main-sidebar {
+  float: right;
+  width: 35%;
+}
+
+.clear {
+  clear: both;
+}
+
+.editor-heading {
+  margin-top: 150px;
+  font-family: helvetica, arial, sans-serif;
+  font-weight: 700;
+  color: rgb(0, 0, 0);
+  padding-bottom: 12px;
+  font-size: 14px;
+  line-height: 14px;
+  letter-spacing: 0.02em;
+  border-bottom: 1px solid rgb(226, 226, 226);
+}
+
+.editor-pics {
+  display: flex;
+  margin: 20px 5px;
+  font-size: 11px;
+  font-weight: 500;
+}
+
+.editor-pics > img {
+  width: 100px;
+  height: auto;
+  margin-right: 10px;
+}
+
+.social-section {
+  display: grid;
+  grid-template-columns: 7fr 1fr 1fr 1fr 1fr 1fr 1fr;
+  margin: 20px 30% 20px 30%;
+}
+
+.article-text {
+  margin: 20px 30% 20px 30%;
+  font-size: 1em;
+  font-family: Georgia, "Times New Roman", Times, serif;
+}
+
+p {
+  display: block;
+}
+
+.social > p,
+.main-w > p {
+  font-size: 0.75em;
+}
+
+.article-text > p,
+.main-article > p {
+  font-size: 21px;
+  line-height: 1.875rem;
+  width: 100%;
+  font-family: "Times New Roman", Times, serif;
+  margin-bottom: 20px;
+  margin-top: 10px;
+}
+
+.article-footer p {
+  font-size: 0.8em;
+}
+
+.article-footer a {
+  font-size: 0.8em;
+  font-weight: normal;
+  text-decoration: underline;
+}
+
+.internal-link a {
+  font-size: 0.7em;
+  color: #666;
+}
+
+.first a {
+  font-size: 1.15em;
+  font-weight: normal;
+  line-height: 1.8em;
+}
+
+.subscriptions a {
+  font-size: 1.2em;
+  font-weight: 800;
+  line-height: 1.4em;
+  position: relative;
+  top: -4px;
+}
+
+.aplications-container a {
+  font-size: 11px;
+}
+
+p > a:hover {
+  text-decoration: underline;
+}
+
+.footer-list li a {
+  font-size: 0.9em;
+  font-weight: normal;
+  color: #666;
+  padding: 5px;
+}
+
+.expand > p > a {
+  font-size: 0.9em;
+}
+
+.article-footer a:hover {
+  text-decoration: none;
+}
+
+.sidebar-line2 a:hover {
+  text-decoration: none;
+  color: #666;
+}
+
+.first a:hover {
+  text-decoration: underline;
+}
+
+.footer-list li a:hover {
+  text-decoration: underline;
+}
+
+.inflation {
+  margin: 0 auto;
+  width: 75%;
+  font-family: georgia, times new roman, times, serif;
+  padding-top: 50px;
+}
+
+.inflation > h1 {
+  line-height: 1.5rem;
+  font-weight: 700;
+  font-style: normal;
+  margin: 0 0 2px 0;
+  color: #000;
+  display: block;
+}
+
+.inflation > p {
+  font-size: 0.9em;
+  color: #666;
+  margin: 0 0 50px 0;
+  line-height: 1rem;
+}
+
+.youtube-iframe {
+  padding: 0;
+  width: 560px;
+  height: 350px;
+  border: none;
+  margin-top: 20px;
+}
+
+.iframecaption {
+  color: rgb(105, 102, 102);
+  font-family: georgia, "times new roman", times, serif;
+  font-size: 0.9375rem !important;
+  line-height: 1rem !important;
```

Comment:
### [OPTIONAL]

- Try not to use `!important` in your CSS. If you need to override a style, try to using selectors with a higher specificity.